packages/forms/frontend/sirius-components-forms/src/representations/FormRepresentation.tsx

```diff
@@ -76,83 +51,37 @@ const useFormRepresentationStyles = makeStyles()((theme) => ({
   },
 }));
 
-/**
- * Connect the Form component to the GraphQL API over Web Socket.
- */
+const isFormRefreshedEventPayload = (payload: GQLFormEventPayload): payload is GQLFormRefreshedEventPayload =>
+  payload && payload.__typename === 'FormRefreshedEventPayload';
+
 export const FormRepresentation = ({ editingContextId, representationId, readOnly }: RepresentationComponentProps) => {
-  const { classes } = useFormRepresentationStyles();
-  const [{ value, context }, dispatch] = useMachine<FormRepresentationContext, FormRepresentationEvent>(
-    formRepresentationMachine,
-    {
-      context: {
-        formId: representationId,
-      },
-    }
-  );
-  const { toast, formRepresentation } = value as SchemaValue;
-  const { id, formId, form, message } = context;
   const [state, setState] = useState<FormRepresentationState>({
-    payload: null,
+    form: null,
   });
 
-  /**
-   * Displays an other form if the selection indicates that we should display another properties view.
-   */
+  const { payload } = useFormSubscription(editingContextId, representationId);
```

Comment:
You are not using the complete information to indicate that the form has been deleted like it was before

Reply:
Done